lib/services/presets.go

```diff
@@ -108,6 +108,7 @@ func NewPresetAccessRole() types.Role {
 				DatabaseNames:        []string{teleport.TraitInternalDBNamesVariable},
 				DatabaseUsers:        []string{teleport.TraitInternalDBUsersVariable},
 				Rules: []types.Rule{
+					types.NewRule(types.KindConnectionDiagnostic, RW()),
```

Comment:
Why does the access role need this rule?

Reply:
My reasoning was: if you can access a resource, we should be able to generate a diagnostic.

Removing it.
We can always add it later on if we see fit.